mopro-core/src/middleware/circom/serialization.rs

```diff
@@ -66,24 +70,30 @@ mod tests {
     use crate::middleware::circom::utils::assert_paths_exists;
     use crate::MoproError;
     use ark_bn254::Bn254;
-    use ark_circom::{CircomBuilder, CircomConfig};
+    use ark_ec::bn::Bn;
     use ark_groth16::Groth16;
     use ark_std::rand::thread_rng;
     use color_eyre::Result;
 
     type GrothBn = Groth16<Bn254>;
 
     fn generate_serializable_proving_key(
-        wasm_path: &str,
         r1cs_path: &str,
     ) -> Result<SerializableProvingKey, MoproError> {
-        assert_paths_exists(wasm_path, r1cs_path)?;
+        assert_paths_exists(r1cs_path)?;
 
-        let cfg = CircomConfig::<Bn254>::new(wasm_path, r1cs_path)
-            .map_err(|e| MoproError::CircomError(e.to_string()))?;
+        // TODO: cleanup the circom-compat implementation
+        // TODO: clean up error handling
+        // why do we have a MoproError type?
```

Comment:
@oskarth

Reply:
Ah that comment is old, I believe the `MoproError` exists for compatibility with Uniffi but could use more clarification.

Reply:
can you help delete the messages here (if it is not used)